src/checkov/checkovRunner.ts

```diff
@@ -33,16 +41,14 @@ const getDockerRunParams = (workspaceRoot: string | undefined, filePath: string,
     const prismaUrlParams = prismaUrl ? ['--env', `PRISMA_API_URL=${prismaUrl}`] : [];
     const debugLogParams = debugLogs ? ['--env', 'LOG_LEVEL=DEBUG'] : [];
 
-    const caCertDockerParams = getDockerFileMountParams(caMountDir, certPath);
-    const caCertCheckovParams = certPath ? ['--ca-certificate', `"${caMountDir}/${path.basename(certPath)}"`] : [];
-
-    const configFileDockerParams = getDockerFileMountParams(configMountDir, configFilePath);
-    const configFileCheckovParams = configFilePath ? ['--config-file', `"${configMountDir}/${path.basename(configFilePath)}"`] : [];
+    const [caCertDockerParams, caCertCheckovParams] = getPathParamsForDockerRun(caMountDir, certPath, '--ca-certificate');
+    const [configFileDockerParams, configFileCheckovParams] = getPathParamsForDockerRun(configMountDir, configFilePath, '--config-file');
+    const [extChecksDockerParams, extChecksCheckovParams] = getPathParamsForDockerRun(extChecksMountDir, extChecksDir, '--external-checks-dir');
```

Comment:
I don't think we should abbreviate to `ext`, `externalChecks` is clearer and more readable IMO
(same applies to all the usage of `ext` in this PR)

Reply:
ok will change